http/handler.go

```diff
@@ -39,6 +41,9 @@ func Handler(core *vault.Core) http.Handler {
 	mux.Handle("/v1/sys/", handleLogical(core, true, nil))
 	mux.Handle("/v1/", handleLogical(core, false, nil))
 
+	// This is probably wrong in quite a few ways - unsure how to register it properly? Something like Mounting, maybe?
```

Comment:
Not sure about go-metrics, but from plain Prometheus this is "the way". Just squat on `/metrics`, unconditionally – because of the pull model, there is nothing to configure and if it's not getting queried you're not paying any price for this.